lib/git_reflow/commands/refresh.rb

```diff
@@ -0,0 +1,21 @@
+desc 'refresh will merge your current branch based on the base branch'
+long_desc <<LONGTIME
+  Performs the following:\n
+  \t$ git checkout <base_branch>\n
+  \t$ git pull <remote_location> <base_branch>\n
+  \t$ git checkout <current_branch>\n
+  \t$ git pull origin <current_branch>\n
+  \t$ git push origin <current_branch>\n
+  \t$ git merge <base_branch>\n
+LONGTIME
+arg_name '[remote_location] - remote repository name to fetch updates from (origin by default), [base_branch] - branch that you want to merge with (master by default)'
+command :refresh do |c|
+  c.action do |global_options, options, args|
+    # usage: git reflow refresh [remote_location - defaults to origin] [base_branch_name - defaults to master]
+    # defaults remote_location to origin
+    remote_location = args.length < 2 ? "origin" : args[1]
+    base_branch = args.length < 1 ? "master" : args[0]
```

Comment:
`git reflow refresh last-feature` only has 1 parameter and the 1st parameter is the `remote_location`, so this command will try to pull from the `last-feature` remote. What you ideally want in this case is `git checkout last-feature` and then `git reflow refresh origin master` in order to sync the last-feature branch with the master branch from remote origin.

We currently only `refresh` the branch that you are currently on and you would have to checkout to that branch yourself.